templates/travel/partials/hero/_hero.css

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@keyframes travel-hero-scene1 {
+  0% {
+    opacity: 0;
+    transform: translate3d(5vmax, 2vmax, 0) scale(1.1);
+  }
+  5% {
+    opacity: 1;
+  }
+  33% {
+    opacity: 1;
+  }
+  38% {
+    opacity: 0;
+    transform: translate3d(0, -2vmax, 0) scale(1);
+  }
+}
+
+@keyframes travel-hero-scene2 {
+  0% {
+    opacity: 0;
+    transform: translate3d(-2vmax, 0vmax, 0) scale(1);
+  }
+  5% {
+    opacity: 1;
+  }
+  33% {
+    opacity: 1;
+  }
+  38% {
+    opacity: 0;
+    transform: translate3d(2vmax, -2vmax, 0) scale(1.1);
+  }
+}
+
+@keyframes travel-hero-scene3 {
+  0% {
+    opacity: 0;
+    transform: translate3d(0, 2vmax, 0) scale(1);
+  }
+  5% {
+    opacity: 1;
+  }
+  33% {
+    opacity: 1;
+  }
+  38% {
+    opacity: 0;
+    transform: translate(0, -2vmax) scale(1.1);
+  }
+}
+
+.travel-hero {
+  height: 100vmax;
+}
+
+.travel-hero-bg {
+  height: 70vmax;
+  animation: fade 0.8s ease-out both;
+}
+
+.travel-hero-bg-img {
+  top: -5vmax;
+  right: -5vmax;
+  bottom: -5vmax;
+  left: -5vmax;
+  opacity: 0;
+  background: var(--silver)
+}
+.travel-hero-bg-img img {
+  object-fit: cover;
+}
+.travel-hero-bg-img:nth-child(3) img {
+  object-position: 80% 100%;
+}
+
+.travel-hero-bg-img:nth-child(1) {
+  animation: 18s travel-hero-scene1 linear infinite;
+  background: url('../../img/travel/hero-2-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-img:nth-child(2) {
+  animation: 18s travel-hero-scene2 linear 6s infinite;
+  background: url('../../img/travel/hero-3-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-img:nth-child(3) {
+  animation: 18s travel-hero-scene3 linear 12s infinite;
+  background: url('../../img/travel/hero-1-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-fade {
+  height: 70vmax;
+  top: calc(70vmax - (300 / 16rem));
+  background-image: linear-gradient(180deg, transparent 0%, var(--black) calc(300 / 16rem));
+}
+
+.travel-hero-content {
+  padding-top: var(--fixed-navbar-height);
+  height: calc(100vmax - 70px);
+}
+
+.travel-hero-heading {
+  color: var(--white);
+  font-size: calc(60 / 16rem);
+  font-weight: 700;
+}
+
+.travel-hero-subheading {
+  color: var(--white);
+  font-size: calc(18 / 16rem);
```

Comment:
most of these calculations need to use variables, it will be super hard for any one to understand what is going on with all these calculations and very easy to break

Reply:
Ok, for whenever we are using the to-rem func `x => calc(x / 16rem)` I will make the call to either make a variable of convert to px.

Reply:
done -
https://github.com/ampproject/ampstart/pull/676/commits/74d7f18e43962338f070f492af030f2f3f5f7227